pkg/kapp/resources/resources.go

```diff
@@ -472,12 +475,20 @@ func (c *ResourcesImpl) resourceErr(err error, action string, resource Resource)
 	return resourcePlainErr{err, action, resource}
 }
 
-func (c *ResourcesImpl) resourceClient(resource Resource) (dynamic.ResourceInterface, ResourceType, error) {
+type resourceClientOpts struct {
+	warnings bool
```

Comment:
even though resourceClientOpts is private, it's field warnings _should_ be public since you are setting it in functions above.

Reply:
Made the changes!